actions/watch.ts

```diff
@@ -268,11 +315,13 @@ function getOrderUid(network: string, orderToSubmit: Order, owner: string) {
  * @param orders All the orders that are being tracked
  */
 export const printUnfilledOrders = (orders: Map<BytesLike, OrderStatus>) => {
-  console.log(`Unfilled orders (${orders.size}):`);
-  for (const [orderUid, status] of orders.entries()) {
-    if (status === OrderStatus.SUBMITTED) {
-      console.log(orderUid);
-    }
+  const unfilledOrders = Array.from(orders.entries())
+    .filter(([_orderUid, status]) => status === OrderStatus.SUBMITTED)
```

Comment:
```suggestion
    .filter(([_orderUid, status]) => status === OrderStatus.SUBMITTED) // as SUBMITTED != FILLED
```

Just a bit of documentation to make it a little more obvious as to why we're filtering on `SUBMITTED`. In hind sight, I should have included.